TestProjects/com.unity.mobile-notifications-sample/Scripts/iOSTest.cs

```diff
@@ -121,9 +121,9 @@ private void InstantiateAllTestButtons()
             {
                 StartCoroutine(GetCurrentLocation());
             });
-            m_groups["General"]["Request Authorization"] = new Action(() =>
+            m_groups["General"]["Request Authorization Sound + Badge + Alert"] = new Action(() =>
```

Comment:
minor, could do 

```
var options = AuthorizationOption.Alert | AuthorizationOption.Badge | AuthorizationOption.Sound;
m_groups["General"]["Request Authorization " + options.ToString()]

...
                StartCoroutine(RequestAuthorization(options));

```

Reply:
The only issue of ToString() is the options are concatenated by ',' rather than '+'. It's ok but not consistent with other buttons. I'll leave QA to make the decision.